.github/workflows/push.yml

```diff
@@ -10,16 +12,29 @@ env:
   TEST_AWS_SECRET_ACCESS_KEY: ${{ secrets.TEST_AWS_SECRET_ACCESS_KEY }}
   TEST_AWS_KMS_KEY_ID: c474f77b-65db-4778-975b-6f167e27d385
   TEST_AWS_ROLE_ID: arn:aws:iam::480319613404:role/ci-object-encryptor-role
+  TEST_GCP_KMS_KEY_ID: projects/kbc-ci-platform-services/locations/global/keyRings/ci-object-encryptor/cryptoKeys/ci-object-encryptor
```

Comment:
Neni tohle to, co si provisionujes pomoci TF?

Reply:
jo

Reply:
Jo pockej :bulb: tady je sice CI provisioning, ale neni integrovany do te pipeline :smile:

Reply:
No dobre, pak teda jo, ale poprosil bych rovnou nahodit issue, aby si to ta pipeline normalne tahala z TF sama, jako jinde https://github.com/keboola/oauth-service/blob/6fccac88227aab9e43b5eb21db1fae07fe88f8b8/.github/workflows/ci.yml#L48

Predpokladam, ze to tu je takle proto, ze v dobe, kdy se pipeline delala, nebyl prokopany TF na GHA.

Reply:
ano, ani tady moc nemám potřebu to dělat, CI resourcy jsou vytvořený, takže doufám, že není důvod na ně příštích 5 let nebude nikdo šahat :) 

save your energy for the deamon :)

Reply:
Ale můžem se k tomu někdy vrátit https://keboola.atlassian.net/browse/PST-1020